pipelines/coaddDiffMatchedInjectedCore.yaml

```diff
@@ -0,0 +1,201 @@
+
+      atools.matchedRefCompleteness: MatchedRefCoaddCompurityTool
+      atools.matchedRefCompleteness.mag_target: psf_err
+      atools.matchedRefCompleteness.mag_bins.mag_low_min: mmag_low_min
+      atools.matchedRefCompleteness.mag_bins.mag_low_max: mmag_low_max
+      atools.matchedRefCompleteness.mag_bins_plot.mag_low_min: mmag_low_min
+      atools.matchedRefCompleteness.mag_bins_plot.mag_low_max: mmag_low_max
+
+      atools.matchedRefAngularSeparationDiff: MatchedRefCoaddDiffDistanceTool
+      atools.matchedRefAngularSeparationDiff.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefAngularSeparationDiff.produce.plot.yLims: lims_dist_diff_full
+
+      atools.matchedRefAngularSeparationDiffZoom: MatchedRefCoaddDiffDistanceTool
+      atools.matchedRefAngularSeparationDiffZoom.produce.plot.xLims: lims_mag_x_zoom
+      atools.matchedRefAngularSeparationDiffZoom.produce.plot.yLims: lims_dist_diff_zoom
+      atools.matchedRefAngularSeparationDiffZoom.produce.metric: NoMetric
+
+      atools.matchedRefAngularSeparationChi: MatchedRefCoaddDiffDistanceTool
+      atools.matchedRefAngularSeparationChi.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefAngularSeparationChi.produce.plot.yLims: lims_dist_chi
+      atools.matchedRefAngularSeparationChi.compute_chi: true
+
+      atools.matchedRefPsfColorDiff: MatchedRefCoaddDiffColorTool
+      atools.matchedRefPsfColorDiff.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPsfColorDiff.produce.plot.yLims: lims_color_diff_full
+
+      atools.matchedRefPsfColorDiffZoom: MatchedRefCoaddDiffColorTool
+      atools.matchedRefPsfColorDiffZoom.produce.plot.xLims: lims_mag_x_zoom
+      atools.matchedRefPsfColorDiffZoom.produce.plot.yLims: lims_color_diff_zoom
+      atools.matchedRefPsfColorDiffZoom.produce.metric: NoMetric
+
+      atools.matchedRefPsfColorChi: MatchedRefCoaddDiffColorTool
+      atools.matchedRefPsfColorChi.compute_chi: true
+      atools.matchedRefPsfColorChi.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPsfColorChi.produce.plot.yLims: lims_mag_chi
+
+      atools.matchedRefPsfMagDiff: MatchedRefCoaddDiffMagTool
+      atools.matchedRefPsfMagDiff.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPsfMagDiff.produce.plot.yLims: lims_mag_diff_full
+
+      atools.matchedRefPsfMagDiffZoom: MatchedRefCoaddDiffMagTool
+      atools.matchedRefPsfMagDiffZoom.produce.plot.xLims: lims_mag_x_zoom
+      atools.matchedRefPsfMagDiffZoom.produce.plot.yLims: lims_mag_diff_zoom
+      atools.matchedRefPsfMagDiffZoom.produce.metric: NoMetric
+
+      atools.matchedRefPsfMagChi: MatchedRefCoaddDiffMagTool
+      atools.matchedRefPsfMagChi.compute_chi: true
+      atools.matchedRefPsfMagChi.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPsfMagChi.produce.plot.yLims: lims_mag_chi
+
+#      atools.matchedRefPositionXDiff: MatchedRefCoaddDiffPositionTool
+#      atools.matchedRefPositionXDiff.coord_meas: x
```

Comment:
Is this meant to all still be in here?

Reply:
It would need the x/y columns to be in the matched table to work. I'm debating whether to add them and make both x/y and ra/dec plots. Perhaps @cmsaunders has thoughts? I figure the difference plots will be uninteresting if the WCS is good (and if not, then it will show up elsewhere), but the chi plots might be a worthwhile sanity check for ra/dec errors.

Reply:
I can understand leaving it for later if it is going to be added back in but then maybe add a note and a TODO to say why it is there.

Reply:
I didn't look through the whole ticket to fully get the context, but if this is going to show ra/dec for the pipeline position versus the injected position, that sounds interesting to me.

Reply:
There will be ra/dec, the question is whether to also make plots of x/y (and particularly chi=(x_meas - x_ref)/x_err).